tests/SampleApps/php/php-fpm-config/NGINX_DUMMY_CUSTOM_CONFIG_FILE.txt

```diff
@@ -0,0 +1,49 @@
+user www-data;
```

Comment:
just so that it more closely represents an expected use case, can we rename this file with a `.conf` extension?